demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -986,6 +986,10 @@ export class DemoMeetingApp
           break;
       }
       try {
+        if (this.chosenVideoTransformDevice) {
+          await this.chosenVideoTransformDevice.stop();
+          this.chosenVideoTransformDevice = null;
+        }
         await this.openVideoInputFromSelection(videoInput.value, true);
```

Comment:
After this call to `openVideoInputFromSelection` we should add a check for the selected video filter and create the transform device. My concern is if we don't do that then the UI and the video stream might not be in sync.